tests/Resources/ODS.robot

```diff
@@ -338,3 +338,23 @@ OpenShift Resource Component Field Should Not Be Empty
     [Arguments]    ${resource_component_field}
     Run Keyword And Continue On Failure    Should Not Be Empty    ${resource_component_field}
 
+
+Reboot Cluster Node
+    [Documentation]    Reboots the specified node of the cluster
+    ...    Args:
+    ...        node(str): Name of the node to reboot
+    ...    Returns:
+    ...        None
+    [Arguments]    ${node}
+    Run    oc debug node/"${node}" -T -- sh -c "echo b > /proc/sysrq-trigger"
+
+Verify ODS Availability
```

Comment:
I'm working in a PR that adds a similar keyword with some enhancements. If your PR gets merged first I'll probably delete this one and adapt your test

Reply:
????